.eslintrc

```diff
@@ -66,7 +70,7 @@
     "no-nested-ternary": 1,
     "no-spaced-func": 1,
     "no-trailing-spaces": 1,
-    "space-after-keywords": [1, "always"],
+    "keyword-spacing": 1,
```

Comment:
:thinking: linting commits in a bugfix PR? I'll allow it.

Reply:
One assumes my editor was complaining...